webclient/src/websocket/utils/sanitizeHtml.util.ts

```diff
@@ -17,13 +17,13 @@ export function sanitizeHtml(msg: string): string {
 }
 
 function enforceTagWhitelist($el: JQuery<HTMLElement>, tags: string): void {
-  $el.find('*').not(tags).each(() => {
+  $el.find('*').not(tags).each(function enforceTag() {
```

Comment:
What's the change about? Same for the other ones below

Reply:
lint rule.